### DISCUSSION TITLE
Tuned Simons' Basic support

### DISCUSSION THREAD

**Discussion by Shadowtrance** [frntc/Sidekick64#67] [Ideas]
Any chance of adding support for this (title)?
Seems pretty cool and would be nice to have it working.
I tried both crt versions available, checking with cartconv...
1 is magic desk and the other is super games. Neither boot. They both do boot on kung fu flash however (but my KFF is having issues so I'd rather use sidekick)

https://github.com/godot64/TSB
https://www.godot64.de/TSB/downloads.html
```
TSB_md.crt
CRT Version: 1.0 (C64 CARTRIDGE )
Name: TSB 2.30220 TUNED SIMONS' BASIC
Hardware ID: 19 (Magic Desk)
Hardware Revision: 0
Mode: exrom: 0 game: 1 (8k Game)

offset  sig  type  bank start size  chunklen
$000040 CHIP ROM   #000 $8000 $2000 $2010
$002050 CHIP ROM   #001 $8000 $2000 $2010
$004060 CHIP ROM   #002 $8000 $2000 $2010
$006070 CHIP ROM   #003 $8000 $2000 $2010
```

```
TSB_up.crt
CRT Version: 1.0 (C64 CARTRIDGE )
Name: 32KB-MODULTEST
Hardware ID: 8 (Super Games)
Hardware Revision: 0
Mode: exrom: 0 game: 0 (16k Game)

offset  sig  type  bank start size  chunklen
$000040 CHIP ROM   #000 $8000 $4000 $4010
$004050 CHIP ROM   #001 $8000 $4000 $4010
$008060 CHIP ROM   #002 $8000 $4000 $4010
$00c070 CHIP ROM   #003 $8000 $4000 $4010
```

Thanks! :)

**Comment by frntc** [comment]
Interesting. In principle it should work -- since it doesn't I should have a look :-) Give me a few days!

**Reply by frntc** [reply]
Found it, will include a fix in the next release!
